packages/core/src/Stores/client-store.js

```diff
@@ -113,20 +113,19 @@ export default class ClientStore extends BaseStore {
     };
 
     @observable account_limits = {};
-    @observable account_limits = {};
-
     @observable self_exclusion = {};
 
     @observable local_currency_config = {
         currency: '',
         decimal_places: undefined,
     };
-    @observable has_cookie_account = false;
 
+    @observable has_cookie_account = false;
+    @observable initial_search_params = '';
```

Comment:
do we have to make this observable? I think it should work fine without observable.

Reply:
right, we don't need it, thank you for pointing this out 😊